src/main/java/org/jabref/logic/importer/fetcher/MathSciNet.java

```diff
@@ -62,20 +65,33 @@ public URL getURLForEntry(BibEntry entry) throws URISyntaxException, MalformedUR
             return getUrlForIdentifier(mrNumberInEntry.get());
         }
 
-        URIBuilder uriBuilder = new URIBuilder("https://mathscinet.ams.org/mrlookup");
-        uriBuilder.addParameter("format", "bibtex");
-
-        entry.getFieldOrAlias(StandardField.TITLE).ifPresent(title -> uriBuilder.addParameter("ti", title));
-        entry.getFieldOrAlias(StandardField.AUTHOR).ifPresent(author -> uriBuilder.addParameter("au", author));
-        entry.getFieldOrAlias(StandardField.JOURNAL).ifPresent(journal -> uriBuilder.addParameter("jrnl", journal));
-        entry.getFieldOrAlias(StandardField.YEAR).ifPresent(year -> uriBuilder.addParameter("year", year));
+        URIBuilder uriBuilder = new URIBuilder("https://mathscinet.ams.org/mathscinet/api/freetools/mrlookup");
+
+        entry.getFieldOrAlias(StandardField.AUTHOR).ifPresentOrElse(
+                author -> uriBuilder.addParameter("author", author),
+                () -> uriBuilder.addParameter("author", "")
+        );
+        entry.getFieldOrAlias(StandardField.TITLE).ifPresentOrElse(
+                title -> uriBuilder.addParameter("title", title),
+                () -> uriBuilder.addParameter("title", "")
+        );
+        entry.getFieldOrAlias(StandardField.JOURNAL).ifPresentOrElse(
+                journal -> uriBuilder.addParameter("journal", journal),
+                () -> uriBuilder.addParameter("journal", "")
+        );
+        entry.getFieldOrAlias(StandardField.YEAR).ifPresentOrElse(
+                year -> uriBuilder.addParameter("year", year),
+                () -> uriBuilder.addParameter("year", "")
+        );
```

Comment:
With a little bit less code:
```suggestion
        uriBuilder.addParameter("author", entry.getFieldOrAlias(StandardField.AUTHOR).orElse(""));
	uriBuilder.addParameter("title", entry.getFieldOrAlias(StandardField.TITLE).orElse(""));
	uriBuilder.addParameter("journal", entry.getFieldOrAlias(StandardField.JOURNAL).orElse(""));
        uriBuilder.addParameter("year", entry.getFieldOrAlias(StandardField.YEAR).orElse(""));
```